### PR TITLE
Remove the serving operator.

### PR DESCRIPTION
**Pull request by markusthoemmes**
The old serving/operator has been replaced by the knative-operator + the upstream serving operator so all this code is no longer needed.

### CODE REVIEW

File: knative-operator/pkg/apis/addtoscheme_obsolete_serving_v1alpha1.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package apis
 
 import (
-	"github.com/openshift-knative/serverless-operator/serving/operator/pkg/apis/serving/v1alpha1"
+	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/apis/serving/v1alpha1"
```
Comment (jcrossley3) [inline knative-operator/pkg/apis/addtoscheme_obsolete_serving_v1alpha1.go]: This is a mistake, isn't it? We need to refer to the old CRD to support migration. That's the what the "obsolete controller" watches.

Comment (markusthoemmes) [reply knative-operator/pkg/apis/addtoscheme_obsolete_serving_v1alpha1.go]: Absolutely, changed the approach.

**Pull request merged**